subworkflows/nf-core/abundance_differential_filter/main.nf

```diff
@@ -49,29 +52,17 @@ workflow ABUNDANCE_DIFFERENTIAL_FILTER {
         .combine(ch_contrasts)
         .multiMap(criteria)
 
-    // We only need a normalised matrix from one contrast. The reason we don't
-    // simply use the first output from DIFFERENTIAL modules is that depending
-    // on the contrast setting etc, these modules may subset matrices, hence
-    // not returning the full normalized matrix as NORM modules would do.
-    norm_inputs = ch_input
```

Comment:
The reason it was done this way is to apply only the first contrast to each combination of count matrix etc from `ch_input`. So unless I'm missing something you've broken it.